packages/fluentui/react/src/themes/teams/components/Accordion/accordionTitleStyles.ts

```diff
@@ -8,6 +8,16 @@ const accordionTitleStyles: ComponentSlotStylesPrepared<AccordionTitleProps> = {
     padding: '.5rem 0',
     cursor: p.disabled ? 'default' : 'pointer'
   }),
+  content: ({ props: p }) => ({
+    display: 'grid',
+    alignItems: 'center',
+    gridTemplateColumns: [
+      'auto', // indicator
+      !!p.content && '1fr'
+    ]
+      .filter(Boolean)
```

Comment:
```tsx
gridTemplateColumns: `auto ${p.content ? '1fr' : ''}`
```

nit: Do we really need arrays magic there?

Reply:
Not really, I just copied the logic from the layoutStyles. It's refactored now